src/NuGet.Clients/PackageManagement.VisualStudio/SolutionRestore/RestoreOperationLogger.cs

```diff
@@ -182,19 +202,16 @@ public void WriteLine(VerbosityLevel verbosity, string format, params object[] a
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            if (OutputVerbosity >= (int)verbosity && _buildOutputPane.Value != null)
+            if (OutputVerbosity >= (int)verbosity && _outputWindowPane != null)
```

Comment:
Can this throw if StartAsync wasn't called? _outputWindowPane could be null for other reasons, but it seems safer to just fail if this wasn't initialized